gapic/src/main/com/google/gapid/views/ErrorDialog.java

```diff
@@ -137,18 +136,13 @@ private void createDetails(Composite container) {
         withLayoutData(createTextbox(inner, DETAILS_STYLE, detailString),
             new GridData(SWT.FILL, SWT.FILL, true, true));
         withLayoutData(createLink(inner, "<a>File a bug</a> on GitHub", e -> {
-          Program.launch(getFileBugUrl());
+          Program.launch(URLs.FILE_BUG_URL);
         }), new GridData(SWT.RIGHT, SWT.BOTTOM, false, false));
         withLayoutData(createLink(inner, "<a>Show logs</a> directory", e -> {
           AboutDialog.showLogDir(analytics);
         }), new GridData(SWT.RIGHT, SWT.BOTTOM, false, false));
       }
 
-      private String getFileBugUrl() {
-        Escaper esc = UrlEscapers.urlFormParameterEscaper();
```

Comment:
nit: remove `import com.google.common.escape.Escaper; import com.google.common.net.UrlEscapers;` together to avoid JAVA warning.